### PR TITLE
remove blit and javadoc i need comments

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

**Review by falseresync — APPROVED** [review]

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 1]
```diff
@@ -36,51 +40,52 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
 		ARG 2 x2
 		ARG 3 y2
 		ARG 4 color
-	METHOD method_25295 innerBlit (IIIIIFFFF)V
-		ARG 0 xStart
-		ARG 1 xEnd
-		ARG 2 yStart
-		ARG 3 yEnd
+	METHOD method_25295 drawTexturedQuad (IIIIIFFFF)V
+		ARG 0 x1
```
Comment (modmuss50) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping]: The old names here were a bit more helpful. I don think they were ideal tho, but better than `x1`.

**Review by jaskarth — COMMENTED** [review]
Looks good apart from the x1/x2 changes

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 1 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: Don't think I have seen those x and xEnd anywhere except here before, also I don't think it doesn't really have to be the start and end of a texture.

Comment (liach) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: These parameters are not part of mojang map, but our improvisations. I feel they are good and have no need to change.

Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: alrighty

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 2]
```diff
@@ -55,32 +59,33 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
 		ARG 8 vEnd
 	METHOD method_25296 fillGradient (IIIIII)V
-		ARG 1 top
-		ARG 2 left
-		ARG 3 right
-		ARG 4 bottom
+		ARG 1 x1
+		ARG 2 y1
+		ARG 3 x2
+		ARG 4 y2
```
Comment (jaskarth) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping]: parameter names not reverted here

**Review by jaskarth — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
@shedaniel remember to update branch merging into to 20w11a later

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 2 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: they were straight out wrong before.

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 3]
```diff
@@ -1,33 +1,37 @@
...
 		ARG 4 v
 		ARG 5 width
 		ARG 6 height
-		ARG 7 texHeight
-		ARG 8 texWidth
+		ARG 7 textureHeight
+		ARG 8 textureWidth
 	METHOD method_25292 drawHorizontalLine (IIII)V
 		ARG 1 x1
 		ARG 2 x2
```
Comment (liach) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping:31]:
```suggestion
		ARG 1 xStart
		ARG 2 xEnd
```

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 4]
```diff
@@ -36,15 +40,15 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
 		ARG 7 vHeight
-		ARG 8 texWidth
-		ARG 9 texHeight
+		ARG 8 textureWidth
+		ARG 9 textureHeight
 	METHOD method_25294 fill (IIIII)V
 		ARG 0 x1
 		ARG 1 y1
 		ARG 2 x2
 		ARG 3 y2
```
Comment (liach) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping:49]:
```suggestion
		ARG 0 xStart
		ARG 1 yStart
		ARG 2 xEnd
		ARG 3 yEnd
```

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 5]
```diff
@@ -36,15 +40,15 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
 		ARG 4 color
-	METHOD method_25295 innerBlit (IIIIIFFFF)V
+	METHOD method_25295 drawTexturedQuad (IIIIIFFFF)V
 		ARG 0 xStart
 		ARG 1 xEnd
 		ARG 2 yStart
```
Comment (liach) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping:54]: Are the xEnd and yStart in right order here?

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]:
```suggestion
		ARG 1 xStart
		ARG 2 yStart
		ARG 3 xEnd
		ARG 4 yEnd
```

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 6]
```diff
@@ -55,32 +59,33 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
+		ARG 2 y1
+		ARG 3 x2
+		ARG 4 y2
 		ARG 5 color1
 		ARG 6 color2
-	METHOD method_25297 innerBlit (IIIIIIIFFII)V
+	METHOD method_25297 drawTexture (IIIIIIIFFII)V
 		ARG 0 xStart
 		ARG 1 xEnd
 		ARG 2 yStart
```
Comment (liach) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping:71]: are the xEnd and yStart swapped here?

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 3 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:31]: No, they are interchangeable, you can swap them around for this one.

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 4 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:49]: No, they are interchangeable, you can swap them around for this one.

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 5 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: Yes.

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 4 above)
Comment (liach) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:49]: then should be good.

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: @shedaniel are they interchangable here?

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 5 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:54]: yes

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 6 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:71]: no

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 2 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping]: oops, forgot to fix this.

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Pull request merged**